plugin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Plugin Name: Meeting Calendar
+ * Description: This provides a way of scheduling recurring meetings, and displaying a calendar or timetable..
+ * Plugin URI: https://github.com/Automattic/meeting-calendar
+ * Author: Automattic
+ * Version: 1.1.0
+ * License: GPL2+
+ * License URI: https://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+function a8c_meeting_calendar_register() {
+
+	// Register our block script with WordPress
+	wp_register_script(
+		'a8c-meeting-calendar',
+		plugins_url('build/index.js', __FILE__),
+		array('wp-blocks', 'wp-element', 'wp-editor')
+	);
+
+	// Register our block's base CSS
+	wp_register_style(
+		'a8c-meeting-calendar-style',
+		plugins_url( 'style.css', __FILE__ )
+	);
+
+	// Register our block's editor-specific CSS
+	wp_register_style(
```

Comment:
This is a personal preference, but I'd remove these style registrations until we have CSS files

Reply:
I realized belatedly that there is a CSS file – we don't need to register the same file twice though, the file registered to `style` will also be loaded in the editor.

Reply:
Yeah, it was a place holder. I'll kill it for now.